doc/changes/latest.inc

```diff
@@ -166,3 +166,5 @@ API changes
 - In `mne.compute_source_morph`, the ``niter_affine`` and ``niter_sdr`` parameters have been replaced by ``niter`` and ``pipeline`` parameters for more consistent and finer-grained control of registration/warping steps and iteration (:gh:`9505` by `Alex Rockhill`_ and `Eric Larson`_)
 
 - Split :func:`mne.viz.Brain.show_view` argument ``view`` into ``azimuth``, ``elevation`` and ``focalpoint`` for clearer view setting and make the default for ``row`` and ``col`` apply to all rows and columns (:gh:`9596` by `Alex Rockhill`_)
+
+- Deprecate :func:`mne.inverse_sparse.mxne_optim._mixed_norm_solver_prox` due to performance reasons: coordinate descent solvers consistently outperforms proximal gradient descent (:gh:`9608` by `Pierre-Antoine Bannier`_)
```

Comment:
Don't mention private functions/methods in latest.inc (this is a user-facing document and users should not have to care about private functions), and also give people a hint of what they need to do to change their code, e.g.:
```suggestion
- Deprecate ``solver='prox'`` in :func:`mne.inverse_sparse.mxne_inverse` in favor of ``solver='cd'`` and ``solver='bcd'`` as coordinate descent solvers consistently outperform proximal gradient descent (:gh:`9608` by `Pierre-Antoine Bannier`_)
```

Reply:
Ok! understood!